.gitignore

```diff
@@ -0,0 +1,15 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
+/.idea/libraries
+/.idea/modules.xml
+/.idea/workspace.xml
+/.idea/navEditor.xml
+/.idea/assetWizardSettings.xml
```

Comment:
@hieuwu I think we should ignore the whole `.idea` folder, as there're still auto-generated files which are not yet ignored in my side.

```suggestion
.idea/
```

![Screenshot 2024-05-01 at 16 57 49](https://github.com/ARK-Builders/ARK-Drop/assets/4281969/13ff9dd8-6df7-462c-aa41-865aa6cc644a)